### ISSUE TITLE
some questions about the results

### ISSUE THREAD

**Issue by youwh-PIRI** [HongwenZhang/PyMAF#75]
First of all, thank you for your excellent work, and I'd like to ask you a few questions
1、When I trained the second stage according to the pare training config, I found that the loss continued to decrease, but the result getting worse
2、The training used eft 、coco pretrain、pare mixdataset setting，But the lspet dataset is not used，the best result of majpe is epoch 0 82.15(pymafx paper 79.7)，Is the difference of lspet dataset so much? 
3、If possible, could you please send me the download link of lspet? Now the official website is invalid
this is my email 2705837989@qq.com，thanks very much!
![image](https://user-images.githubusercontent.com/48080676/205044746-780f602f-35ee-4064-b513-30c9cce8d38b.png)

**Comment by HongwenZhang** [comment]
Hi, thank you for your questions.
1) The result gets worse because the data distribution of 3DPW differs with Human3.6M. So the decreasing loss means that the model is better fitted to the Human3.6M dataset but not 3DPW.
2) Is the Spatial Alignment Attention used in your experiments? The majpe of 79.7 is the performance of PyMAF with Spatial Alignment Attention. The lspet dataset may not be the main reason for the performance gap.
3) I will send you the link in a couple of days. 
Thanks!

**Comment by youwh-PIRI** [comment]
Hi, thank you for your training advice, now I got the approximate results on 3dpw(majpe 80), but there are many poor results on human3.6. May I ask if there are any additional training skills when testing human3.6
